staging/src/k8s.io/api/events/v1/types.go

```diff
@@ -23,6 +23,7 @@ import (
 
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +k8s:prerelease-lifecycle-gen:introduced=1.19
```

Comment:
Event should be 1.0?

Reply:
This was added because the PR adding this struct was in 1.19 milestone

https://github.com/kubernetes/kubernetes/pull/91645